src/pages/_app.js

```diff
@@ -1,5 +1,15 @@
 import "@/styles/globals.css";
+import { Roboto } from '@next/font/google'
+
+const roboto = Roboto ({
+  subsets: ['latin'],
+  weight: ['400','700']
+})
 
 export default function App({ Component, pageProps }) {
-  return <Component {...pageProps} />;
+  return (
+    <main className={roboto.className}>
```

Comment:
<img width="520" alt="image" src="https://github.com/Sanket-Deb/portfolio-website/assets/69194538/33ce7527-4327-485c-8eb1-a19dc0399c3b">
Why is **main** tag used to wrap the components ?
You already have a main tag index.js 
Will end up rendering main twice like the above image.